java/spacewalk-java.changes

```diff
@@ -1,3 +1,6 @@
+- Fixed a bug that caused the tab Autoinstallation to hide when clicking on Power 
```

Comment:
Nitpick: That's a very log message, make it concise that will be useful for the user when they will read it. Moreover, if there is bugzilla issue behind it, you need to add the reference for that bsc.

Reply:
I will shorten the message @admd ! Also no, there's no bugzilla reference whatsoever. Just discovered the bug while developing a new feature.